source/summary.md

```diff
@@ -0,0 +1,425 @@
+
+> [!NOTE]
+> _Some context on auditing and systematic evaluation_: It is important to note that audits, when performed true to their intent, are about focusing on gathering evidence of failures, issues, gaps, or problems.
+> While simply auditing can then establish a direct line between evidence of a failure and opening an issue on GitHub, we used auditing as an opportunity to take a step back. Our evidence serves as a basis for an analysis of systemic and interconnected issues across Bokeh's ecosystem.
+> What separates this kind of high-level audit from the typical atomic or incremental approach most commonly used in open source software (which is often ad hoc) is that we have a chance to evaluate and analyze an entire ecosystem and synthesize all possible related problems at once.
+> Addressing incremental problems as they arise can have limits, especially if there are significantly interconnected issues. As an example, it is common to see an issue on an open source repo that asks for "alt text capabilities." While this might be great to add on, sometimes a library or tool needs several larger refactors or overhauls to really reach ideals/goals and uphold principles of good design. Data visualization libraries in particular have a habit of simply implementing ARIA directly on chart elements to address a problem like "alt text" but fail to recognize that this easily sets up new and more complex accessibility barriers down the road.
+> In order to do a more systematic evaluation (and even know what the larger issues are), having a procedure for gathering evidence, taking a step back, finding similarities, connecting the biggest gaps, and setting goalposts is vital. Consider this document as a demonstration of this process.
+
+### <a id="credits" href="#credits" aria-label="Credits"><span aria-hidden="true">#</span></a> Credits
+Big collaborations like this involve many different kinds of labor. We use the [Contributor Role Taxonomy (CRediT) Classification](https://credit.niso.org/) system to acknowledge the specific work each team member did to contribute to our success.
+
+- [**Frank Elavsky**](https://www.frank.computer/), CMU, [@frankelavsky](https://github.com/frankelavsky): writing - original draft preparation (lead), data curation (lead), formal analysis (lead), investigation (lead), methodology (lead), conceptualization (supporting), project administration (supporting), and software (supporting).
+- **Tania Allard**, Quansight, [@trallard](https://github.com/trallard): conceptualization (lead), funding acquisition (lead), resources (lead), project administration (lead), supervision (lead), writing - review and editing (lead), software (supporting), visualization (supporting).
```

Comment:
- feel free to suggest any change here, if you'd like!

Reply:
This looks good; we can update later if needed, thank you.